changelog.md

```diff
@@ -1,6 +1,21 @@
 ## Changes from 0.14.0 to 0.15.0
 
-### Breaking Changes
+### Recommended Mesos version is 0.26.0
+
+We tested this release against Mesos version 0.26.0. Thus, this is the recommended Mesos version for this
+release.
+
+### Overview
+
+#### Integration of Mesos Fetcher Cache
+
+The v2 REST API was extended to support the Mesos fetcher cache. This allows users to configure a list of resource URIs
+that will be copied into the task sandbox prior to running the task, from either a local or external location. For
+details on the fetcher cache's capabilities, please see the fetcher cache [documentation](http://mesos.apache.org/documentation/latest/fetcher/).
+
+#### Migration from Marathon version 0.5 removed
+
+It is no longer possible to migrate from Marathon version 0.5.
```

Comment:
Maybe add a note that says that you need to upgrade to intermediary versions as is anyway recommended.